docs/reference/commandline/attach.md

```diff
@@ -14,6 +14,7 @@ parent = "smn_cli"
 
     Attach to a running container
 
+      --detach-keys       Specify the escape key sequence used to detach a container
```

Comment:
Hmm, the way I read @vdemeester comment the `ctrl-` was hardcoded but the follow on value was not. That's what the code looks like. Let's see what he says, I can tweak the language.